both/methods.js

```diff
@@ -65,6 +65,11 @@ setSolveTime = function(resultId, solveIndex, solveTime) {
   RoundSorter.addRoundToSort(result.roundId);
 };
 
+function getTodayDateNoTime() {
+  var today = moment();
+  return new Date(Date.UTC(today.year(), today.month(), today.date()));
```

Comment:
So I think if you do **getTodayDateNoTime()** in the client, and pass the result with the createCompetition call, you would get the correct "today" for the timezone on the users computer?

I agree on the short term/long term priorities.